README.md

```diff
@@ -1 +1,263 @@
-# angular-inheriting-or-isolate-scope-readme
\ No newline at end of file
+# Inheriting or isolating scope
+
+## Overview
+
+So far, our directives we've made aren't really that useful. We're not doing anything special - let's change that!
+
+## Objectives
+
+- Describe scope inheritance
+- Describe scope isolate
+- Create an inherited scope
+- Create an isolate scope
+- Bind data to the scope
+- Access inherited and isolate scope properties
+
+## Scope inheritance
+
+Now, one cool things about our directives is that they can display dynamic data. For instance, we might want to have a Twitter card to display a users Twitter handle, along with a link to follow said user.
+
+We can do this exactly like we've done in our views so far - using `{{}}`.
+
+Note: now that we've started using more HTML in our directives, instead of just passing a string, we pass in a multi-line array, joined by an empty string. This means we can have a multi-lined template without having to worry about string concatenation.
+
+Lets create the Twitter card.
+
+```js
+function TwitterCard() {
+	return {
+		template: [
+			'<div class="twitter">',
+				'<a href="https://twitter.com/">Follow @username on twitter!</a>',
+			'</div>'
+		].join(''),
+		restrict: 'E'
+	};
+}
+
+angular
+	.module('app')
+	.directive('twitterCard', TwitterCard);
+```
+
+Now that's cool, but how do we *actually* get our data? Well, we can actually inherit it from the scope above.
+
+Let's assume we've got a controller, that has a property on it's scope named `twitter`. We would display this in our view as follows:
+
+```html
+<div ng-controller="SomeController">
+	{{ twitter }}
+</div>
+```
+
+If we were to use our Twitter card directive instead of displaying `{{ twitter }}`, we could actually use `{{ twitter }}` in our directives template.
+
+```html
+<div ng-controller="SomeController">
+	<twitter-card></twitter-card>
+</div>
+```
+
+```js
+function TwitterCard() {
+	return {
+		template: [
+			'<div class="twitter">',
+				'<a href="https://twitter.com/{{ twitter }}">Follow @{{ twitter }} on Twitter!</a>',
+			'</div>'
+		].join(''),
+		restrict: 'E'
+	};
+}
+
+angular
+	.module('app')
+	.directive('twitterCard', TwitterCard);
+```
+
+This will result in this HTML being put into the DOM (we're assuming `$scope.twitter` equals `billgates`.)
```

Comment:
Is it worthwhile to show an example using ControllerAs? If the "real way" to do it is to use isolate scope, that example's not important. If you might want to use ControllerAs here for some reason, probably good to include an example.